src/main/java/igrad/ui/MainWindow.java

```diff
@@ -253,6 +254,25 @@ void show() {
         primaryStage.show();
     }
 
+    /**
+     * Displays the sad version of the avatar when loading
+     */
+    private void handleLoading(Avatar avatar) {
+
+        Avatar sadAvatar = new Avatar(avatar.getName() + "-sad");
```

Comment:
Please take a look at how `refreshStatusBar(..)` is done, i don't think you  should be  calling `Avatar` from here, but instead use `model.getAvatar...`  something.  use the  `Model` public API interface to access Avatar don't access it directly here  (breaking abstraction/SLAP)